java/code/src/com/redhat/rhn/frontend/servlets/CreateRedirectURI.java

```diff
@@ -14,13 +14,12 @@
  */
 package com.redhat.rhn.frontend.servlets;
 
-import com.redhat.rhn.frontend.action.LoginAction;
+import com.redhat.rhn.frontend.action.LoginHelper;
```

Comment:
If `LoginAction` has turned into a `LoginHelper` class, is the `com.redhat.rhn.frontend.action` package still the appropriate place for this class?

Reply:
Moved into another package